### ISSUE TITLE
Mobile App Report

### ISSUE THREAD

**Issue by shreya-mishra** [RealDevSquad/mobile-app#266]
**P1** 
- [ ] Fix the Test Issue - @shreya-mishra 
- [ ] CI/CD Flow diagram and RFC  - @shreya-mishra 
- [ ] POC for CI/CD
- [ ] CI/CD Implementation  - @shreya-mishra 
- [x] Decide the Priority between Goals , notify and fixing QR flow (replace multiple clicks via realtime exchanging response while scanning) - @ankushdharkar 
- [x] Clear the Requirement of Notify If Notify app is in priority - https://github.com/Real-Dev-Squad/mobile-app/issues/265 - @ankushdharkar 
- [x] Spike ticket for Notify service.
- [x] RFC for notify service.
- [ ] POC for notify service.
- [ ] Implementation the notify in rds app.
- [ ] enhancement of notify service.
- [ ] Brainstorming on QR Flow and document it - @shreya-mishra 
- [ ] Brainstorming on how do we share builds directly from discord, soon will be sharing document.
- [x] Raise the tickets of Notify as well as QR flow when above will be verified via Ankush (**Blocker**)
- [ ] API contract for goals backend - @pallabez 
- [ ] Goals app ticket Raise : **Blocker - need api contract - **P0****
- [x] Goals TODOs : https://github.com/Real-Dev-Squad/mobile-app/issues/288 : 

**Things which is done/progress (P2)**

- [x] SignIn with github 
- [x] SignIn with QR basic flow
- [x] HomeScreen where we can check the status of the user and post the request for OOO (in staging it will get merge to develop with in 2 days)
- [x] Profile screen with their contributions (depends on backend for the cookie once Kotesh PR merge it will work fine)
- [ ] Working on Profile screen management where all the contributions should have shown - @JAYATEJA04 
- [ ] Implementing Redux - in progress by @mohammadkhizerkhan 

**Definition of Priority**
P0 – the site is down and all work stops, until this issue is resolved.
P1 – finish this task to unblock someone else, required to be done before other things
P2 – ordinary flow of work
P3 – nice to have, but not required
P4 – informational only

**Comment by shreya-mishra** [comment]
1. goals App is the priority as discussed !
2. Notification app
